### DISCUSSION TITLE
Welcome to MIUIwidgets Discussions!

### DISCUSSION THREAD

**Discussion by R9TRH** [R9TRH/MIUIwidgets#2] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by R9TRH** [comment]
Hello & welcome

**Reply by joeyoropesa-dev** [reply]
Thank you so much 🥰🥰🥰

**Comment by joeyoropesa-dev** [comment]
One question, if I want to disable temporary or remove this module will I be able after I got an system update to bring back stock system apps that is replaced by the module

Because, I've tested this on my own device just to boot into core-only mode without modules but my device crashed and launcher and other apps that is being replaced by this module didn't exist in the system anymore (including the stock one)

And every file I've replaced by custom GApps via Magisk module is permanently replaced in the system so when I've tried to remove it, the system didn't bring back my lost system files and apps about Google system and it's goes into bootloop

So, what can I do

**Reply by R9TRH** [reply]
Hello
I'm happy that I still have communication with my mod users this time by GitHub

For uninstalling any magisk module without having issues and traces in system partition, Uninstall that module (not disable, core mode etc.) And before reboot, flash package cache  cleaner. I will attach it in next reply, so all issues you mentioned should be solved

**Reply by R9TRH** [reply]
[Packagecachecleaner.zip](https://github.com/R9TRH/MIUIwidgets/files/10469974/Packagecachecleaner.zip)

**Reply by joeyoropesa-dev** [reply]
> [Packagecachecleaner.zip](https://github.com/R9TRH/MIUIwidgets/files/10469974/Packagecachecleaner.zip)
> 

Yes, it worked for me - this is bring back my removed system apps and files thats is replaced before due to some module - thank you so much, you saved my phone to not be soft/hard-bricked 💖🫂🥰

**Comment by Vader77Darth** [comment]
Does this module work on Xiaomi Pad 5. I know that the Xiaomi pads come with a different launcher: release 4.40.x version

**Reply by R9TRH** [reply]
Probably no
Do you have any source for pad apps' updates?
Then I can release versions for pad users

**Reply by Vader77Darth** [reply]
APKMirror has them I believe. The RELEASE 4.40.x versions are the ones for tablets specifically. https://www.apkmirror.com/apk/xiaomi-inc/miui-launcher/miui-launcher-release-4-40-0-5305-11181053-release/ This is the latest one and Miui updated me to that one automatically.

**Reply by R9TRH** [reply]
Thank you very much
A release for pad devices will be uploaded shortly
Are Themes, App vault and search apps exactly same with mobile? Just for confirming.

Also,
Is there any telegram channel for pads?
My friend John Imaginer can share me apps from telegram

**Reply by Vader77Darth** [reply]
I have tried the launcherless version and themes, app vault, search worked (couldn't add widgets though because incompatible system launcher) The tablets actually disabled the theme store (can only change wallpaper in settings.) but your module installs the chinese theme store which for some reason is enabled so that's good. I don't know of any telegram channels for Xiaomi Pads but there could be some that I do not know of.

**Reply by R9TRH** [reply]
Sounds good
Thanks for helping me in developing and fixing bugs of project👍

**Reply by Vader77Darth** [reply]
You're Welcome!

**Reply by R9TRH** [reply]
Are pathes for Xiaomi Pads same as Phones?
I'm Making first V8 series for Pads and need to confirm paths.

**Reply by Vader77Darth** [reply]
No. Launcher is in /system/priv-app/MiuiHomePadS/

**Reply by R9TRH** [reply]
Oh ok
Thanks

**Reply by R9TRH** [reply]
Looks like there is no Big Folder and Super Icon option in latest Pad launcher. I couldn't find the code inside app.
So don't report for that in release.

**Reply by Vader77Darth** [reply]
Ok.

**Reply by R9TRH** [reply]
Did V8 Beta work on your device?

**Reply by R9TRH** [reply]
I do not think so, bcz I didn't find code of picker activity on global codes to replace with china ones. I will wait for your report

**Reply by Vader77Darth** [reply]
I installed it. It added a search button which I cannot disable. Please let me disable it. The widgets menu works and I had to open themes first to make it work but there is no themes on the the homescreen so I had to launch it from settings which worked. There was no app vault until I enabled app vault by launching "-1 screen" in activity launcher which added an app vault. But, I cannot add widgets to the app vault. When I do, they just appear dead small and don't work.

**Reply by Vader77Darth** [reply]
Also, the new releases are still completely broken on Poco phones.

**Reply by R9TRH** [reply]
It is weird that you can not disable search box...
It is somehow normal that you had to launch themes and app vault manually bcz the pad didn't support it by default
Does adding new widgets to homescreen work?

Also, there are many Pocos running this module+addon well. What is your Poco ROM?

**Reply by Vader77Darth** [reply]
Adding widgets to the home screen works fine.
Miui 13.0.4.0 SJGEUXM

**Reply by R9TRH** [reply]
Message my friend in Telegram to send you BETA2 for pad devices
T.me/JohnImaginer

**Reply by Vader77Darth** [reply]
Ok.

**Reply by Vader77Darth** [reply]
The new version isn't working for me. The system launcher is gone. not even installed. Also, the EU v7 series never worked for me before.
